Incorporate GAT feedback (#1922)
galaxyproject/training-material: topics/admin/tutorials/external-auth/tutorial.md

>
>    `GX_SECRET` is added as a header for security purposes, to prevent any other users on the system impersonating nginx and sending requests to Galaxy. NGINX and other webservers like Apache will strip any user-sent `REMOTE_USER` headers, as that header defines the authenticated user. If you can talk directly to Galaxy (e.g. via curl) and provide the `REMOTE_USER` header, you can impersonate any other use. While having Galaxy listen on `127.0.0.1` prevents any requests from outside of the system reaching Galaxy, anyone on the system can still send requests to that port. Here you can choose to switch to a unix socket with permissions only permitting Galaxy and Nginx to connect. `GX_SECRET` adds additional security as it needs to match `remote_user_secret` in your galaxy configutation.
>
>    > ### {% icon tip %} Proxy bypass

> >    > Users can bypass and only if they can talk directly to the uwsgi processes (if you have socket/http: 0.0.0.0, or if it is directly responsible for serving galaxy, and there is no proxy.)

Reviewer: ```suggestion
>    > Users can bypass the authentication only if they can talk directly to the uWSGI processes (if you have socket/http: 0.0.0.0, or if it is directly responsible for serving galaxy, and there is no proxy.)
```

> >    > When this can happen, mostly it is due to users who have command line access to the Galaxy server which is usually a bad practice.

Reviewer: ```suggestion
>    > This can happen mostly when some users have command line access to the Galaxy server, which is considered a bad practice.
```
